.gitignore

```diff
@@ -37,9 +37,13 @@ variables*.yml
 *ansible/inventory/*
 .env
 /tests/metrics/results/
-/docs/build
 .python-version
 *.log
 *no datastore
 finnegans-wake.txt
-*venvs
\ No newline at end of file
+*venvs
+
+# Sphinx
+docs/build
+docs/source/modules.rst
```

Comment:
Seems like this file is no more...?

Reply:
I'm working on it - mis-configured apidoc.

Reply:
:heavy_check_mark: